app/models/docker_container_wizard_states/exposed_port.rb

```diff
@@ -0,0 +1,11 @@
+module DockerContainerWizardStates
+  class ExposedPort < Parameter
+    belongs_to :environment,  :foreign_key => :reference_id, :inverse_of => :exposed_ports,
+                              :class_name => 'DockerContainerWizardStates::Environment'
+    validates :name,  :uniqueness => { :scope => :reference_id },
+                      :numericality => { :only_integer => true }
```

Comment:
Try creating a port with a string, say 'aaaa', for some reason this works. Also please validate for inclusion in 1-65535